detector/aws_security_group_duplicate_name.go

```diff
@@ -0,0 +1,86 @@
+package detector
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/wata727/tflint/issue"
+)
+
+type AwsSecurityGroupDuplicateDetector struct {
+	*Detector
+}
+
+func (d *Detector) CreateAwsSecurityGroupDuplicateDetector() *AwsSecurityGroupDuplicateDetector {
+	return &AwsSecurityGroupDuplicateDetector{d}
+}
+
+func (d *AwsSecurityGroupDuplicateDetector) Detect(issues *[]*issue.Issue) {
```

Comment:
__[gometalinter]__
* [gocyclo] cyclomatic complexity 17 of function (*AwsSecurityGroupDuplicateDetector).Detect() is high (> 15)

<!-- Comment by [SideCI](https://sideci.com) -->